wear/src/main/java/com/teamcasey/watchdog/WatchdogWatchFace.java

```diff
@@ -0,0 +1,475 @@
+                        textPaintForDrawing
+                );
+            } else {
+                canvas.drawText(
+                        "Rows: " + this.currentRowCount,
+                        xOffsetForDrawing,
+                        yOffsetForDrawing + (textSpacingHeightForDrawing * 3),
+                        textPaintForDrawing
+                );
+            }
+            */
+
+            /** Covers area under peek card */
+            if (isInAmbientMode()) {
+                canvas.drawRect(cardBounds, peekCardBackgroundPaintForDrawing);
+            }
+        }
+
+        /**
+         * Store the heartrate change and prompt the user to relax if above their baseline
+         *
+         * @param event from API
+         */
+        @Override
+        public void onSensorChanged(SensorEvent event) {
+            if (event.values[0] > 0) {
+                this.currentHeartRate = (int) event.values[0];
+                new InsertHeartRateRow().execute((int) event.values[0]);
+
+                invalidate();
+
+                if (this.hasEnoughRowsForBaseline() && this.currentHeartRate >= this.baselineHeartRate) {
+                    if (this.recordsUntilPrompt == 0) {
+                        this.askUserIfTheyWishToRelax();
+                        this.recordsUntilPrompt = 10;
+                    } else {
+                        this.recordsUntilPrompt--;
+                    }
+                } else {
+                    this.recordsUntilPrompt = 10;
+                }
+            }
+        }
+
+        @Override
+        public void onAccuracyChanged(Sensor sensor, int accuracy) {
+            // Filler, needs to be implemented
+        }
+
+        /**
+         * Runs whenever the watch face becomes visible/invisible
+         *
+         * @param visible -> true if visible false otherwise
+         */
+        @Override
+        public void onVisibilityChanged(boolean visible) {
+            super.onVisibilityChanged(visible);
+
+            if (visible) {
+                setupHeartRateMonitoring();
+            } else {
+                disableHeartRateMonitoring();
+            }
+        }
+
+        /**
+         * Creates the sensors and starts listening for heart rate changes
+         */
+        private void setupHeartRateMonitoring() {
+            this.sensorManager = ((SensorManager) getSystemService(SENSOR_SERVICE));
+            this.heartRateSensor = sensorManager.getDefaultSensor(Sensor.TYPE_HEART_RATE);
+
+            this.sensorManager.registerListener(this, this.heartRateSensor, 1);
+        }
+
+        /**
+         * Shuts the hr sensor off
+         */
+        private void disableHeartRateMonitoring() {
+            this.sensorManager.unregisterListener(this);
+        }
+
+        /**
+         * Creates an alert dialog asking if the user wishes to relax
+         */
+        private void askUserIfTheyWishToRelax() {
```

Comment:
In this function is where we need to launch the watchdog application and get the user to start interacting with other apps